cmd/mailchain/internal/settings/root.go

```diff
@@ -30,6 +30,8 @@ func FromStore(s values.Store) *Root {
 			}),
 			protocols.Substrate: protocol(s, protocols.Substrate, map[string]NetworkClient{
 				substrate.EdgewareTestnet: network(s, protocols.Substrate, substrate.EdgewareTestnet, defaults.SubstrateNetworkAny()),
+				substrate.PolkadotTestnet: network(s, protocols.Substrate, substrate.PolkadotTestnet, defaults.SubstrateNetworkAny()),
+				substrate.KusamaTestnet:   network(s, protocols.Substrate, substrate.KusamaTestnet, defaults.SubstrateNetworkAny()),
```

Comment:
Mailchain requires the contract module to be loaded, currently only EdgewareTestnet has this loaded. Can you these lines, in the future we will be able add them back in or others